randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -947,7 +955,7 @@ Extra - room_id: [25]
           Combat (Beginner) or 20 DMG Missiles
           Any of the following:
               # Normal DMG
-              Normal Damage ≥ 200
+              Normal Damage ≥ 170
               # Fight Boss Damageless: https://youtu.be/WdEd1majcTU
               Damage Boosts (Intermediate)
```

Comment:
not a part of the diff, but why is this damage boost? assume this should be combat (int)

Reply:
Good question, it should indeed be combat. Ill go fix it too